homeassistant/components/config/entity_registry.py

```diff
@@ -152,6 +152,15 @@ def websocket_get_entities(
         # If passed in, we update value. Passing None will remove old value.
         vol.Optional("aliases"): list,
         vol.Optional("area_id"): vol.Any(str, None),
+        # Categories is a mapping of key/value (scope/category_id) pairs.
+        # If passed in, we update/adjust only the provided scope(s).
+        # Other category scopes in the entity, are left as is.
+        #
+        # Categories can only be in 1 category ID per scope at a time.
+        # Therefore, passing in a category ID will either add or move
+        # the entity to that specific category. Passing in None will
+        # remove the entity from the category.
+        vol.Optional("categories"): cv.schema_with_slug_keys(vol.Any(str, None)),
```

Comment:
How can an entity be in different categories? Shouldn't it only be able to be in the category for the scope that is the same as the entity domain?

Reply:
Depends on the view/dashboard.

From the entities dashboard, you could have a different categorization compared to your automations or zones dashboard.

The original PR (#93498) was based on a `domain`, but we changed it to `scope` as it doesn't have to be a domain-limited view (like, for example, an entities dashboard or whatever else).